src/Plugin/DataType/StrawberryValuesViaJmesPathFromJson.php

```diff
@@ -93,12 +98,21 @@ public function process($langcode = NULL)
         foreach ($values as $value) {
           $result = $parser->parse($value);
           if ($result->isValid()) {
-            $date_ini = $result->getEdtfValue()->getMin();
-            $date_max = $result->getEdtfValue()->getMax();
-            $date_ini = date('c', $date_ini);
-            $date_max = date('c', $date_max);
-            $values_parsed[] = $date_ini;
-            $values_parsed[] = $date_max;
+            $edtf_value = $result->getEdtfValue();
+            switch(get_class($edtf_value)) {
+              case "EDTF\Model\Interval":
```

Comment:
Could you add a small //@TODO remove once EDTF fixes their invalid Constructor for EDTF\Model\Interval that should per interface never allow NULL for start nor end date